README.md

```diff
@@ -42,7 +42,7 @@ module "ses_domain" {
   source             = "trussworks/ses-domain/aws"
   domain_name        = "example.com"
   mail_from_domain   = "email.example.com"
-  route53_zone_id    = data.aws_route53_zone.ses_domain.zone_id
+  route53_zone_id    = data.aws_route53_zone.SES_domain.zone_id
```

Comment:
Could you actually just rename the route53 data source to be `ses_domain`?

Reply:
@dynamike done